.github/workflows/ci_android.yml

```diff
@@ -42,11 +42,16 @@ jobs:
           compiler="armv7a"
         fi
 
+        rdft_opt="--enable-rdft"
+        if [ ${{ matrix.ffmpeg_version }} == "6.1" ]; then
+            rdft_opt=""
+        fi
+
```

Comment:
I believe you can avoid that by extending the matrix like that:
```yaml
      matrix:
        arch: [x86_64, arm, aarch64]
        ffmpeg_version: ["4.4", "6.1"]
        include:
           - ffmpeg_version: 4.4
             ff_flags: "--enable-rdft"
```

See https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs#expanding-or-adding-matrix-configurations

Reply:
Done.